panel/src/components/Dialogs/FormDialog.vue

```diff
@@ -53,6 +59,28 @@ export default {
         return {};
       }
     }
+  },
+  data() {
+    return {
+      model: this.value
+    }
+  },
+  computed: {
+    listeners() {
+      return {
+        ...this.$listeners,
```

Comment:
This is a dark pattern that Vue 3 won't support anymore, so maybe at least for all new code we can already try to find a better implementation?

Reply:
What is the suggested alternative?

Reply:
You could try binding `$attr` instead in Vue 3: https://v3.vuejs.org/guide/migration/listeners-removed.html#_2-x-syntax

But I think the actual reading is that one should clearly define what events and listeners are supported

Reply:
Yeah, it's probably the best idea to be more explicit.

Reply:
Will also help to understand our code better: what events do we actually want to support there?

Reply:
✅